src/platform/forms-system/src/js/utilities/ui/index.js

```diff
@@ -18,6 +18,19 @@ export function focusElement(selectorOrElement, options) {
   }
 }
 
+// Focus on review row _after_ the content has been updated
+export function focusOnChange(name) {
+  setTimeout(() => {
+    const el = document.querySelector(`[name="${name}ScrollElement"]`);
+    // nextElementSibling = page form
+    const targets = el?.nextElementSibling?.querySelectorAll('.review-row');
+    if (targets) {
+      targets.forEach(target => target.setAttribute('tabindex', '0'));
```

Comment:
Would it be possible to [set `tabindex` in the markup](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms-system/src/js/review/ReviewFieldTemplate.jsx#L22) instead?

Reply:
I should probably ask @jenstrickland about this...

Right now, the user can't tab through the review row elements. Initially, I set the `tabIndex` to `-1` which allows focusing, but that's only good if there is one review row. If a page has multiple rows (e.g. mailing address), then it might be better for the user to tab through all the rows.

I set the code in this PR to allow tabbing through the all the review rows, but only for the recently edited form.

So yes, we could add a `tabIndex` to all review rows, but I'm not sure if that's a good idea.

Reply:
@1Copenut — since you're more familiar with the codebase and patterns, would you please respond to @Mottie and @bkjohnson ? Grazi!

Reply:
> but only for the recently edited form.

Is this identified by the `name` argument passed to `focusOnChange`?

Also, re-reading the PR description gave me another question.

>once a user opens an accordion and edits an entry, focus becomes lost

Would it make more sense if we modified the `<Accordion>` component so that it doesn't blur when clicked?  Or is this a misunderstanding of how things are working?

Reply:
`name` is actually the page key from the uiSchema. This code uses it to target the scroll Element for the page, then jump to the next element sibling which is the div that wraps the review rows

Reply:
The rows should not be focusable when the user has exited the the form edit. Rows are a well-structured description list, so they're navigable as they are. I'm attaching 3 screenshots for where focus should go, and some guidance. These are very low-fi, with arrows and fuchsia text. Holler at me with any questions.

---
![Screen Shot 2020-03-12 at 12 14 07 PM](https://user-images.githubusercontent.com/934879/76548163-54c22580-645c-11ea-8aad-ce82e4f6c72f.png)

---
![Screen Shot 2020-03-12 at 12 15 03 PM](https://user-images.githubusercontent.com/934879/76548177-5a1f7000-645c-11ea-8c0d-8a98addbe8da.png)

---
![Screen Shot 2020-03-12 at 12 15 36 PM](https://user-images.githubusercontent.com/934879/76548255-77543e80-645c-11ea-8145-394bfb808cbc.png)

Reply:
Thanks @1Copenut! I'll have these changes included shortly